src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -317,18 +319,31 @@ private void ChooseActions(in Entity entity, ref MicrobeAI ai, ref CompoundAbsor
             return;
         }
 
+        float atpLevel = compounds.GetCompoundAmount(Compound.ATP);
+
         // If this microbe is out of ATP, pick an amount of time to rest
-        if (compounds.GetCompoundAmount(Compound.ATP) < 1.0f)
+        if (atpLevel < 1.0f)
         {
             // Keep the maximum at 95% full, as there is flickering when near full
             ai.ATPThreshold = 0.95f * speciesFocus / Constants.MAX_SPECIES_FOCUS;
         }
 
         if (ai.ATPThreshold > MathUtils.EPSILON)
         {
-            if (compounds.GetCompoundAmount(Compound.ATP) <
-                compounds.GetCapacityForCompound(Compound.ATP) * ai.ATPThreshold)
+            if (atpLevel < compounds.GetCapacityForCompound(Compound.ATP) * ai.ATPThreshold)
             {
+                if (cellHealth.CurrentHealth > 2 * Constants.ENGULF_NO_ATP_DAMAGE)
+                {
+                    // even if we are out of ATP and there is microbe nearby, engulf them.
```

Comment:
```suggestion
                    // Even if we are out of ATP and there is microbe nearby, engulf them.
```

Comments should be started with a capital letter. I'll comment just this once but I saw this elsewhere as well).